apps/crn-server/test/controllers/news.test.ts

```diff
@@ -120,6 +120,164 @@ describe('News controller', () => {
 
       expect(result.items[0]!.thumbnail).toBeUndefined();
     });
+
+    test('Should query data properly when request does not have options', async () => {
+      nock(baseUrl)
+        .get(`/api/content/${appName}/news-and-events`)
+        .query({
+          q: JSON.stringify({
+            take: 8,
+            filter: { path: 'data.type.iv', op: 'ne', value: 'Tutorial' },
+            sort: [{ order: 'descending', path: 'created' }],
+          }),
+        })
+        .reply(200, newsSquidexApiResponse);
+
+      const result = await news.fetch();
+
+      expect(result).toEqual(listNewsResponse);
+    });
+
+    describe('Frequency Filter', () => {
+      test('Should query data properly when News Articles is selected', async () => {
```

Comment:
i think you want to say 
```suggestion
      test('Should query data properly when CRN Quarterly and News Articles frequency is selected', async () => {
```